client/src/components/SitterListing/SitterListing.tsx

```diff
@@ -0,0 +1,53 @@
+import { Box, Typography, Avatar, Paper, Container } from '@material-ui/core';
+import useStyles from './useStyles';
+import { Profile } from '../../interface/Profile';
+import Rating from '@material-ui/lab/Rating';
+import Divider from '@material-ui/core/Divider';
+import LocationOnIcon from '@material-ui/icons/LocationOn';
+import { Link } from 'react-router-dom';
+import { useSitters } from '../../context/useSitterContext';
+import { MouseEvent } from 'react';
+interface Props {
+  profile: Profile;
+}
+
+const SitterListing = ({ profile }: Props): JSX.Element => {
+  const classes = useStyles();
+  const { selectSitter } = useSitters();
+  const handleSelect = (e?: MouseEvent<HTMLElement, MouseEvent<Element, MouseEvent>>) => {
+    selectSitter(profile);
+  };
+  return (
+    <Container className={classes.cardMain} component={Paper}>
```

Comment:
It is a good practice to use the [Card API](https://mui.com/api/card/) when a component contains information about a specific topic :)

Reply:
Thank you for pointing this out. I will definitely look to use the Card API in the future.